pkg/agent/controller/egress/egress_controller_test.go

```diff
@@ -1118,7 +1118,7 @@ func TestSyncEgress(t *testing.T) {
 			if tt.newLocalIPs != nil {
 				c.localIPDetector = &fakeLocalIPDetector{localIPs: tt.newLocalIPs}
 			}
-			assert.NoError(t, wait.Poll(time.Millisecond*100, time.Second, func() (done bool, err error) {
+			assert.NoError(t, wait.PollUntilContextTimeout(context.TODO(), time.Millisecond*100, time.Second, false, func(ctx context.Context) (done bool, err error) {
```

Comment:
did you see my earlier comment: https://github.com/antrea-io/antrea/pull/5843#discussion_r1445462333?

Reply:
Sorry I forget to reply it, I saw it and I'm working on this part. Thanks for the reminder.